pkg/parser/validate/validate_test.go

```diff
@@ -18,15 +18,27 @@ type ValidateTestCase struct {
 }
 
 func CheckYamlErrors(t *testing.T, testCases []ValidateTestCase) {
+	context := &utils.LsContext{
+		Api: utils.ApiContext{
+			Token:   "XXXXXXXXXXXX",
+			HostUrl: "https://circleci.com",
+		},
+	}
 	for _, tt := range testCases {
 		t.Run(tt.Name, func(t *testing.T) {
 			content := tt.YamlContent
-			doc, err := parser.GetParsedYAMLWithContent([]byte(content))
+			doc, err := parser.ParseContent([]byte(content), context)
 			assert.Nil(t, err)
 			val := Validate{
 				Diagnostics: &[]protocol.Diagnostic{},
 				Cache:       utils.CreateCache(),
 				Doc:         doc,
+				Context: &utils.LsContext{
+					Api: utils.ApiContext{
+						Token:   "",
+						HostUrl: utils.CIRCLE_CI_APP_HOST_URL,
+					},
+				},
```

Comment:
why are we using this one here and another one above (line 21) in the same function ?